src/main/java/com/welovecoding/nbeditorconfig/listener/ProjectChangeListener.java

```diff
@@ -25,45 +25,47 @@ public ProjectChangeListener(Project project) {
   }
 
   @Override
+  public void fileFolderCreated(FileEvent event) {
+    super.fileFolderCreated(event);
+    LOG.log(Level.INFO, "PROJECTLISTENER: Created folder: {0}", event.getFile().getPath());
```

Comment:
We should stick to one type of printing console information. At the moment we have 3 different types... One is `[PREFIX] Action: Value`, another one is `\u00ac Action: Value` and now you introduced `PREFIX: Action: Value`.